script/release/uploaders/upload.py

```diff
@@ -23,7 +23,8 @@
                      SRC_DIR, ELECTRON_DIR, TS_NODE
 
 
-ELECTRON_VERSION = get_electron_version()
+ELECTRON_VERSION = 'v' + get_electron_version()
+print(ELECTRON_VERSION)
```

Comment:
do we need to print it?

Reply:
We sure don't, that was me making sure it worked 🙃 Thanks, will remove!